pkg/azure/gw-machineset.go

```diff
@@ -44,6 +44,7 @@ spec:
       metadata:
         labels:
           submariner.io/gateway: "true"
+          submariner.io/machine-set-owner: "submariner"
```

Comment:
As discussed on the call, lets get rid of this new label as its introducing lot of corner cases.

Reply:
Done.